### PR TITLE
Update documentation to stop telling people to use the manifest color icon in seller dashboard.

### PR DESCRIPTION
**Pull request by MattSFT**

### CODE REVIEW

**Comment by charlesprakash** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[msteams-platform/publishing/office-store-guidance.md](https://github.com/MattSFT/msteams-docs/blob/patch-2/msteams-platform/publishing/office-store-guidance.md) | :warning:Warning | | [Details](#user-content-msteams-platform%2Fpublishing%2Foffice-store-guidance.md)

<a name="msteams-platform%2Fpublishing%2Foffice-store-guidance.md"></a>
### [msteams-platform/publishing/office-store-guidance.md](https://github.com/MattSFT/msteams-docs/blob/patch-2/msteams-platform/publishing/office-store-guidance.md)
 - **Line 9**: **[Warning]** Invalid file link:(~/publishing/apps-publish).
 - **Line 73**: **[Warning]** Invalid file link:(~/publishing/apps-publish#microsoft-teams-app-approval-process).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C3%5C30%5Cd98cc23d-94e8-765b-0855-bdd8964dc537%5CPullRequest%5C201803301915083871-128%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=Ds%2B5LvAyo%2BYa3ltGIXu%2FFVTa1j%2BtOwkGt5S2HQF8Aes%3D&st=2018-03-30T19%3A11%3A11Z&se=2018-04-30T19%3A16%3A11Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by charlesprakash** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[msteams-platform/publishing/office-store-checklist.md](https://github.com/MattSFT/msteams-docs/blob/patch-2/msteams-platform/publishing/office-store-checklist.md) | :warning:Warning | | [Details](#user-content-msteams-platform%2Fpublishing%2Foffice-store-checklist.md)
[msteams-platform/publishing/office-store-guidance.md](https://github.com/MattSFT/msteams-docs/blob/patch-2/msteams-platform/publishing/office-store-guidance.md) | :warning:Warning | | [Details](#user-content-msteams-platform%2Fpublishing%2Foffice-store-guidance.md)

<a name="msteams-platform%2Fpublishing%2Foffice-store-checklist.md"></a>
### [msteams-platform/publishing/office-store-checklist.md](https://github.com/MattSFT/msteams-docs/blob/patch-2/msteams-platform/publishing/office-store-checklist.md)
 - **Line 38**: **[Warning]** Invalid file link:(~/concepts/bots/bots-create).

<a name="msteams-platform%2Fpublishing%2Foffice-store-guidance.md"></a>
### [msteams-platform/publishing/office-store-guidance.md](https://github.com/MattSFT/msteams-docs/blob/patch-2/msteams-platform/publishing/office-store-guidance.md)
 - **Line 9**: **[Warning]** Invalid file link:(~/publishing/apps-publish).
 - **Line 73**: **[Warning]** Invalid file link:(~/publishing/apps-publish#microsoft-teams-app-approval-process).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C3%5C30%5Cd98cc23d-94e8-765b-0855-bdd8964dc537%5CPullRequest%5C201803301916378869-128%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=SlO%2FGF3OQgMAVJxHIzv3%2BuuHiL3TVk%2B3IKRr4lwlppA%3D&st=2018-03-30T19%3A12%3A40Z&se=2018-04-30T19%3A17%3A40Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: msteams-platform/publishing/office-store-guidance.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ In the Seller Dashboard:
...
    * **My app calls, support, contains, or uses cryptography or encryption** – This checkbox helps to ensure that your product is not using cryptography in a way that would prohibit distribution through the Office Store. See [Export restrictions on cryptography](https://docs.microsoft.com/en-us/windows/uwp/security/export-restrictions-on-cryptography) for a detailed explanation.
    * **My app calls, support, contains, or uses cryptography or encryption** – This checkbox helps to ensure taht your product is not using cryptography in a way that would prohibit distribution through AppSource. See [Export restrictions on cryptography](https://docs.microsoft.com/en-us/windows/uwp/security/export-restrictions-on-cryptography) for a detailed explanation.
-   * **App logo** – Use your full-color 192&times;192 icon.
+   * **App logo** – Use the icon you want to appear in the AppSource catalog. The Teams gallery will use the icon from your manifest.
```
Comment (msftdherron) [inline msteams-platform/publishing/office-store-guidance.md]: Might be slightly more clear to say 

Use the icon you want to appear in the AppSource catalog. (Note that the gallery within the Teams app will use the icon from your manifest.)

**Comment by charlesprakash** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[msteams-platform/publishing/office-store-checklist.md](https://github.com/MattSFT/msteams-docs/blob/patch-2/msteams-platform/publishing/office-store-checklist.md) | :warning:Warning | | [Details](#user-content-msteams-platform%2Fpublishing%2Foffice-store-checklist.md)
[msteams-platform/publishing/office-store-guidance.md](https://github.com/MattSFT/msteams-docs/blob/patch-2/msteams-platform/publishing/office-store-guidance.md) | :warning:Warning | | [Details](#user-content-msteams-platform%2Fpublishing%2Foffice-store-guidance.md)

<a name="msteams-platform%2Fpublishing%2Foffice-store-checklist.md"></a>
### [msteams-platform/publishing/office-store-checklist.md](https://github.com/MattSFT/msteams-docs/blob/patch-2/msteams-platform/publishing/office-store-checklist.md)
 - **Line 38**: **[Warning]** Invalid file link:(~/concepts/bots/bots-create).

<a name="msteams-platform%2Fpublishing%2Foffice-store-guidance.md"></a>
### [msteams-platform/publishing/office-store-guidance.md](https://github.com/MattSFT/msteams-docs/blob/patch-2/msteams-platform/publishing/office-store-guidance.md)
 - **Line 9**: **[Warning]** Invalid file link:(~/publishing/apps-publish).
 - **Line 73**: **[Warning]** Invalid file link:(~/publishing/apps-publish#microsoft-teams-app-approval-process).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C3%5C30%5Cd98cc23d-94e8-765b-0855-bdd8964dc537%5CPullRequest%5C201803301959063124-128%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=eKHBH2kWiwnEGgbxPVliElDISkImcsHJeeqNAlgi8%2F0%3D&st=2018-03-30T19%3A55%3A09Z&se=2018-04-30T20%3A00%3A09Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**